.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+name: Run All Tests
+
+on:
+  pull_request:
+    types: [
+      opened,
+      labeled,
+      ready_for_review,
+      reopened,
+      synchronize,
+    ]
+    branches:
+      - main
+      - release**
+      - production**
+      - hotfix**
+      - lab**
```

Comment:
@kavimaluskam These are the branch which will run the test

Reply:
Yes as pre-discussed today.
Will do review tmr.

Reply:
Take your time😉